charts/veritable-ui/Chart.yaml

```diff
@@ -0,0 +1,28 @@
+annotations:
+  category: veritable
+  licenses: Apache-2.0
+apiVersion: v2
+appVersion: v0.3.7
+dependencies:
+  - condition: postgresql.enabled
+    name: postgresql
+    repository: https://charts.bitnami.com/bitnami
+    version: 15.x.x
+  - name: common
+    repository: https://charts.bitnami.com/bitnami
+    tags:
+      - bitnami-common
+    version: 2.x.x
+description: The veritable user interface for managing certificates
+home: https://github.com/digicatapult/veritable-ui
+icon: https://raw.githubusercontent.com/digicatapult/sqnc-documentation/main/assets/icon.png
```

Comment:
Check to see if we have an icon for this in `veritable-documentation` if not, remove this line.

Reply:
ok, will confirm on this shortly

Reply:
it does not exists and I've removed the "icon" property all together